integration-tests/cli/helpers/products-export.data.js

```diff
@@ -0,0 +1,194 @@
+export const sampleProductType = {
+  name: 'productTypeForProductExport',
+  description: 'bla bla bla',
+  key: `productTypeKey${Date.now()}`,
+}
+
+export const sampleTaxCategory = {
+  name: 'new-sample-tax-category',
+  key: `tax-cat-1${Date.now()}`,
```

Comment:
it looks like this file is used for testing purposes.
I would use a hard coded value here instead of `Date.now()`

Reply:
I think this is to ensure that the key is unique. Can we bypass, but I think this is okay. No hard opinion.

Reply:
This is to ensure the key is unique. In edge cases, not doing so could cause the tests to fail